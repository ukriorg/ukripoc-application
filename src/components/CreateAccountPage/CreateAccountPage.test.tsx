import React from "react";
import renderer from "react-test-renderer";
import { CreateAccountPage } from "./index";

describe("CreateAccountPage", () => {
    it("renders correctly", () => {
        const tree = renderer.create(<CreateAccountPage />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});

