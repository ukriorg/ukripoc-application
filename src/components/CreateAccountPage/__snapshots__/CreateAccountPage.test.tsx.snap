// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateAccountPage renders correctly 1`] = `
<span
  className="sc-bdVaJa NUDvK"
  size="XL"
>
  Create account
</span>
`;
